src/reducers/resourceReducer.js

```diff
@@ -22,6 +22,27 @@ export function isFetchingResource(state = initialState.isFetchingResource, acti
     }
 }
 
+export function headCategories(state = initialState.headCategories, action) {
+  switch (action.type) {
```

Comment:
Would be good to avoid duplication & let `headCategories()` & `categories()` to share code by extracting a method

Reply:
could be another PR